src/app/docs/how-to/mock-service/mock-service.component.html

```diff
@@ -0,0 +1,16 @@
+<h1>Mocking a Service</h1>
+<section markdown>
+Playground supports mocking service dependencies of application components. Take the following `MyComponent` example
+below that currently has a `LoadDataService` dependency (<i>service included below for brevity</i>).
+</section>
+<div markdown path="/content/docs/mock-service/component.md"></div>
+<section markdown>
+With `LoadDataService` called during the component's `ngOnInit` lifecycle hook (`this.dataService.loadData`), the
+service must be available when the component is initialized or Angular will throw a runtime error. The same occurs for
+sandboxes missing mocked service dependencies.
+
+An easy way to mock a service in a sandbox is to include a mock service class (`MockLoadDataService`) in your
```

Comment:
CHANGE TO:
An easy way to mock a service in a sandbox is to include a mock service class (`MockLoadDataService`) in your
`*.sandbox.ts` file that returns mock data in the format you expect it. Then use Angular's dependency injection to swap out the LoadDataService with the MockLoadDataService.  When the sandboxed component's `ngOnInit` lifecycle hook is called the MockLoadDataService's "loadData()" method will now be called instead of the LoadDataService's "loadData()" method.

Reply:
Updated to include information on dependency injection.